helm/flowforge/templates/service-ingress.yaml

```
@@ -47,3 +47,43 @@ spec:
     - {{ $forgeHostname }}
     secretName: {{ $forgeHostname }}
   {{- end }}
+{{- if gt (int .Values.forge.replicas) 1 -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: flowforge-ingress
```

Ingress objects name must be unique within the namespace/
```suggestion
  name: flowforge-ingress-api-devices
```